dask/bytes/tests/test_http.py

```diff
@@ -177,3 +177,15 @@ def test_bag():
     b = db.read_text(urls)
     assert b.npartitions == 2
     b.compute()
+
+
+@pytest.mark.xfail(reason="https://github.com/dask/dask/pull/5231")
```

Comment:
Can you make this depend on `fsspec.__version__ < 0.4.1`. Then if fsspec 0.4.2 is released and it's still broken, or it breaks again in the future, we'll catch it.

Reply:
Done